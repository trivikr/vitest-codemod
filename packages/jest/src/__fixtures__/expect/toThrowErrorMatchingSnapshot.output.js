import { expect, test } from "vitest";
test("toThrowErrorMatchingSnapshot", () => {
  expect(() => {
    throw new Error("error");
  }).toThrowErrorMatchingSnapshot();
});