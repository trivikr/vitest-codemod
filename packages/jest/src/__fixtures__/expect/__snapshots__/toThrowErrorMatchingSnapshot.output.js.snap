// Vitest Snapshot v1

exports[`toThrowErrorMatchingSnapshot 1`] = `"error"`;
